api/app.go

```diff
@@ -53,6 +54,7 @@ func NewApp(
 		Debug:               debug,
 		Logger:              logger,
 		EmailDomain:         config.GetStringSlice("oauth.acceptedDomains"),
+		AppsRoutesDomain:    config.GetString("kubernetes.appsDomain"),
```

Comment:
so, I was thinking now...
I think that controller itself does not need to have this information, why? Because the CLI can infer this based on the controllerUrl used to login, right? I think we only need controller to return the routable apps and cli can do the rest, what do you think @Henrod ?